Yoojin-week6

```diff
@@ -0,0 +1,93 @@
+// 문자열 출력하기
+
+#include <stdio.h>
+#include <string.h>
+#define LEN_INPUT 1000001
+
+int main(void) {
+    char s1[LEN_INPUT];
+    scanf("%s", s1);
+
+    int len = strlen(s1);
+    for (int i = 0; i < len; i++){
+        printf("%c", s1[i]);
+    }
+    printf("\n");
+    return 0;
+}
+
+// --------------------------------------------------
+
+// a와 b 출력하기
+
+#include <stdio.h>
+
+int main(void) {
+    int a, b;
+    scanf("%d %d", &a, &b);
+    
+    printf("a = %d \nb = %d\n", a, b);
+
+    return 0;
+}
+
+// ------------------------------------------------
+
+// 문자열 반복해서 출력하기
+
+#include <stdio.h>
+#define LEN_INPUT 11
+
+int main(void) {
+    char s1[LEN_INPUT];
+    int a;
+    scanf("%s %d", s1, &a);
+
+    for (int i = 0; i < a; i++) {
+        printf("%s", s1);
+    }
+    printf("\n");
+
+    return 0;
+}
+
+// -----------------------------------------------
+
+// 대소문자 바꿔서 출력하기
+
+#include <stdio.h>
+#include <string.h>
+#define LEN_INPUT 10
+
+int main(void) {
+    char s1[LEN_INPUT];
+    scanf("%s", s1);
+
+    int len = strlen(s1);
```

Comment:
I'm so impressive to use strlen function!

Reply:
Yeah I think that's useful func